From: Aurelien Jarno <aurelien@aurel32.net>
Date: Mon, 1 May 2017 15:58:31 +0200
Subject: glibc default source

_SVID_SOURCE and _BSD_SOURCE are deprecated, using _DEFAULT_SOURCE
instead.
---
 Make-config.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Make-config.in b/Make-config.in
index f0e0a12..8041327 100644
--- a/Make-config.in
+++ b/Make-config.in
@@ -48,7 +48,7 @@ INSTALL_DATA = @INSTALL_DATA@
 INSTALL_SCRIPT = @INSTALL_SCRIPT@
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
 
-CFLAGSD = -D_SVID_SOURCE -D_BSD_SOURCE -DSHM -DHAVE_CONFIG_H
+CFLAGSD = -D_DEFAULT_SOURCE -DSHM -DHAVE_CONFIG_H
 CXXFLAGS =  @CXXFLAGS@ @CPPFLAGS@ ${CFLAGSD}
 CFLAGS   =  @CFLAGS@ ${CFLAGSD}
 
